test/statsd_stress/parameters.yml

```diff
@@ -0,0 +1,66 @@
+# Receivers that agent needs to tests
+receivers: ["statsd"]
+# Processors that agent needs to tests
+processors: []
+# Exporters that agent needs to tests
+exporters: []
```

Comment:
Yes. It will works. The receivers/processors/exporters are only are referring plugins and does not generate from YAML or TOML but input directly. However, it is used to know what metrics load should we refer to.